library/js/oeUI/headerTitleAction.js

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Expand Contract State and Show /Hide State js
```

Comment:
place a .php at the end of this or else developers will get confused (for example, they may try to bring it in as a pure js script which will break):
headerTitleAction.js.php

Reply:
can you explain why the php is getting parsed correctly in the js file ?

Reply:
I looked into this, and it is because you are including it within a php file as a php file (using the require call). So, this is why the apache/nginx don't get involved. Still, though, this will thoroughly confuse developers.